modules/document_repository/css/document_repository.css

```diff
@@ -298,6 +298,7 @@ label em {
     padding-left: 30px;
     height:100%;
     overflow: auto;
+    word-break: break-word;
```

Comment:
break words to remove overflow in x for files. This also fixes the issue with the random scroll bars on the side of files and directories